app/src/main/java/com/example/soundcloud/home_screen/HomeFragmentPagerAdapter.java

```diff
@@ -0,0 +1,41 @@
+package com.example.soundcloud.home_screen;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.example.soundcloud.data.TabInfo;
+import com.example.soundcloud.discover_screen.DiscoverFragment;
+
+public class HomeFragmentPagerAdapter extends FragmentPagerAdapter {
+    private final int PAGE_COUNT = 3;
```

Comment:
hằng số a khai báo staic final 
```suggestion
    private static final int PAGE_COUNT = 3;
```